bbb/services.py

```diff
@@ -236,14 +236,14 @@ def _handleFinishedRequest(self, brid, properties, results):
             #
             # In all cases we need to delete the BuildRequest from our own
             # database.
-            log.info("Marking task %s as cancelled", taskid)
+            log.info("buildrequest %s: task %s: Marking task as cancelled", brid, taskid)
             status = self.tc_queue.status(taskid)["status"]["runs"][runid]
             # If the Task is still running on Taskcluster, cancel it.
             if status.get("state") == "running":
                 self.tc_queue.cancelTask(taskid)
             self.bbb_db.deleteBuildRequest(brid)
         else:
-            log.info("WEIRD: Got unknown results %s, ignoring it...", results)
+            log.info("buildrequest %s: task %s: WEIRD: Got unknown results %s, ignoring it...", brid, taskid, results)
```

Comment:
nit: "WEIRD: (ignoring) recieved unknown results: %s" so we can treat until EOL as the results without confusion ;-)